synapse/lib/stormtypes.py

```diff
@@ -5250,6 +5257,15 @@ async def list(self):
     def value(self):
         return dict(self.valu.props)
 
+    async def _methNodePropsDel(self, name):
```

Comment:
Would we want this to use the newer assign `$lib.undef` paradigm since it already uses the getter/setter paradigm?